storages/json/dlite-json-storage.c

```diff
@@ -190,7 +190,8 @@ DLiteInstance *json_load(const DLiteStorage *s, const char *id)
     buf = jstore_get(js->jstore, uuid);
   }
   if (!buf && !(buf = jstore_get(js->jstore, id)))
-    FAIL2("no instance with id \"%s\" in storage \"%s\"", id, s->location);
+      goto fail;
+    //FAIL2("no instance with id \"%s\" in storage \"%s\"", id, s->location);
```

Comment:
I guess that you commented this out to get rid of many annoying false error messages. 

I hope the branch `global-error-state` will provide a proper fix, but until it is a good idea to keep this commented out.